### ISSUE TITLE
'Reception dates of paper form' adjustments for German Sormas

### ISSUE THREAD

**Issue by max-hzi** [SORMAS-Foundation/SORMAS-Project#2348]
### Feature Description
The way this aspect is designed in Sormas is useless for the German system. We would like to change it to 'Format der Meldung' (format of the report). 

### Proposed Change
I already adjusted the title in the crowdin tool ('Reception dates of paper form' now translates to 'Format der Meldung'). To make an actual change, please introduce a new drop-down menu „Format der Meldung“ with the following entries: Fax, Brief, Telefon, Digital, andere. In case 'andere' is selected, provide a free text box.
The existing date entry fields should remain untouched.

**Comment by max-hzi** [comment]
Ok, small update. I just talked to @bernardsilenou and the conclusion of the call is that we should entirely hide the section 'Reception dates of paper form' in the German Sormas.

The new drop-down menu should be added as suggested above, but with an independent date entry field. These fields should be added to the Case Information/ case tab and above the general comment field (practically superseding the now hidden 'Reception dates of paper form'.

Also raising attention to #1186

**Comment by bernardsilenou** [comment]
@MateStrysewskeSym @markusmann-vg  
Based on the numerous need and issues to hide fields that are not needed on the different instances of SORMAS. I think we need to consider implementing #1186 in our short term road map

**Comment by ChristopherRiedelSymeda** [comment]
@max-hzi 
I understand why the new field is useful for Germany and the old fields are not. But translations should never be changed in the first place. First of all, there will be a separate caption for the new field, which can be translated as "Format der Meldung". Second, it is not yet clear when this ticket will be implemented, probably not in this sprint. This in turn means that in the next release the old date fields will still be there, but with a new caption that makes no sense at all, which will lead to confusion and misunderstandings.

**Comment by max-hzi** [comment]
Yes, I agree, I changed the translation back already. When did you pull the translations? We also had other problems, i. e. other not authorized people used the tool and messed with some translations which we also took care of already.

**Comment by ChristopherRiedelSymeda** [comment]
As far as I know @fhauptmann takes care of the changes in crowdin.

**Comment by bernardsilenou** [comment]
@max-hzi Crowdin is blocked now, only authorized people can translate.
I can allso see that all traslators can translate for all the languages. Maybe we need some restrictions at this level.
@max-hzi maybe send me an email of those who should be allowed to translate to German so that I can block all the others
